docs/new-feature-tutorial.md

```diff
@@ -11,12 +11,44 @@ real-time push system; the
 [real-time push and events](events-system.html) documentation has a
 detailed explanation of how everything works.
 
-## General Process in brief
+## General Process
+
+### Files impacted
+
+This tutorial will walk through adding a new feature to a Realm (an
+organization in Zulip). The following files are involved in the process:
+
+**Backend**
```

Comment:
I'd perhaps reorder these around the flow.  E.g. first the model, then view, then actions, then events at the end.